bugbot/rules/file_crash_bug.py

```diff
@@ -86,6 +86,18 @@ def get_bugs(self, date):
             "Firefox", "nightly"
         )
         signatures = data_fetcher.analyze()
+        # This is the last filtering stage which aims to avoid filling bugs for
+        # junky crashes, where the volume is low and the crashes do not show
+        # signals of being actionable or critical.
+        signatures = [
+            signature
+            for signature in signatures
+            if signature.num_installs > 20
```

Comment:
If this or the next line apply but the ones below not, crash filing will often be delayed.

Reply:
Are you suggesting decreasing these thresholds? Do you have a number in mind?

Reply:
1. Goal (?): catch frequent crashes early: The 10pm UTC Nightly very rarely gets 20 crashing installations, even for the 10am one this takes hour for top crashers (because of the users not updating immediately when the update becomes available and it is common the crash is not on startup).
2. Goal: catch all crashes above a certain frequency threshold in a week: By default, we have 12-14 nightly builds per week. If each update is installed by a machine, every one of them will count as install. Some machines reporting the same signature with 4+ Nightly builds is not uncommon.

Crash signatures for a week can be seen e.g. [here](https://crash-stats.mozilla.org/topcrashers/?product=Firefox&version=120.0a1&_report_type=any&process_type=any&days=7) (modify the version number in the url bar if necessary).

Suggestions (might be future work):
1. Lower threshold to 5 installs (accepting false positive bug reports based on 2. from above).
2. Run the analyzer with two time frames: 7 days and e.g. 1 day, with different thresholds.

Reply:
I lowered the thresholds in 67b0fe83ef50d90c040f69d9681a77cfae4d5bc2